app/views/email_authentications/passwords/expired_reset_token.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="error-body">
+  <div class="dialog">
+    <img src="http://simple.org/images/simple-wordmark-standard.svg">
+    <div class="text">
+      <p class="title">Reset link expired</p>
+      <p>
+        The password link expired. <%= link_to "Click here", new_email_authentication_password_path %> to request a new link.
+      </p>
+    </div>
+    <%= link_to "RETURN TO LOGIN PAGE", "/", class: "return-link" %>
```

Comment:
Should this be all caps?

Reply:
I'll double check with Claudio but I believe that was the request.